umh-core/pkg/communicator/actions/edit-dataflowcomponent.go

```diff
@@ -184,6 +187,15 @@ func (a *EditDataflowComponentAction) Parse(payload interface{}) error {
 		return errors.New("missing required field Name")
 	}
 
+	a.state = models.ComponentState(topLevel.State)
+	if a.state == "" {
```

Comment:
I am not sure if this should be the default error handling. if an invalid state is selected by the user, it should be refused immediately

Reply:
also doesn't make sense that we then afterwards check if its valid, if we always set it right?